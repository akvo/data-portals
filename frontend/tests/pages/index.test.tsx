import React from 'react'
import { render } from '@testing-library/react'
import HomePage from '../../pages/index'

describe('Home page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<HomePage />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
