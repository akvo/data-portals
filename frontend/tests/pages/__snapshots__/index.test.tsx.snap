// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <h1>
    Welcome to Next.js!
  </h1>
  <a
    href="/ping"
  >
    Ping backend
  </a>
</DocumentFragment>
`;
